app/src/main/java/com/skyyo/samples/features/pagination/paging/CatsPagingScreen.kt

```diff
@@ -45,11 +40,11 @@ import kotlinx.coroutines.flow.receiveAsFlow
 fun CatsPagingScreen(viewModel: CatsPagingViewModel = hiltViewModel()) {
     val context = LocalContext.current
     val lifecycleOwner = LocalLifecycleOwner.current
-    val insets = LocalWindowInsets.current
+    val insetsStatusBars = WindowInsets.statusBars
     val density = LocalDensity.current
 
     val insetTop: Dp = remember {
-        with(density) { insets.statusBars.top.toDp() + 8.dp }
+        with(density) { insetsStatusBars.getTop(density).toDp() + 8.dp }
     }
```

Comment:
can become this:
```kotlin
 val insetTop: Dp = remember {
        insetsStatusBars.getTop(density).dp
    }
```
Also let's please do it in other places as well. We will spend some time in optimising code in our samples in the future, so we can start with insets right in this PR